recipe/meta.yaml

```diff
@@ -17,20 +18,24 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - pytest-runner
+    - wheel
     - setuptools >=30.0.0
   run:
     - python >=3.6
     - mccabe >=0.6.0,<0.7.0
-    - pycodestyle >=2.7.0,<2.8.0
-    - pyflakes >=2.3.0,<2.4.0
-    - importlib-metadata  # [py<38]
+    - pycodestyle
+    - pyflakes
+    - importlib-metadata
```

Comment:
Eliminate the unnecesary pinnings

Reply:
We should remove selectors and not pinning. It was my mistake

Reply:
The correct one is:
    - pycodestyle >=2.8.0,<2.9.0
    - pyflakes >=2.4.0,<2.5.0
    - importlib-metadata